### ISSUE TITLE
Geolocation shouldn't depend on Cloudflare

### ISSUE THREAD

**Issue by ZiRo-** [infinity-next/infinity-next#226]
If you want people to use this software, some of them probably don't want to use Cloudflare. The geoip library of the old infinity software should at least be included as a fallback.

**Comment by milezteg** [comment]
Agree. The Maxmind database would be sufficient to determine geolocation:  http://dev.maxmind.com/geoip/geoip2/geolite2/

**Comment by jaw-sh** [comment]
The way I've set it up allows for multiple methods of detection but I do 
not have the time to set up this database right now. Cloudflare probably 
uses the exact same thing.
